### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by adrianheine**

### CODE REVIEW

File: src/edit/selection.js [hunk 1]
```diff
@@ -174,14 +174,14 @@ export class Selection {
   // :: number #path=Selection.prototype.to
   // The right-bound of the selection.
 
-  // :: bool #path=Selection.empty
+  // :: bool #path=Selection.prototype.empty
```
Comment (marijnh) [inline src/edit/selection.js]: This is a property of the constructor, not its prototype.

Comment (marijnh) [inline src/edit/selection.js]: Errr, nope it isn't. Ignore that comment. Was thinking of `Fragment.empty`.

**Comment by marijnh** [comment]
Thank you. Merged as 9a91e73dda32b4...9a22f7e0a3f0e6 .

Did you see [this](http://flowtype.org/blog/2015/02/20/Flow-Comments.html)? It's still a little cumbersome, but it seems a step in the direction of having type information without having a compilation step.

**Comment by adrianheine** [comment]
Yes, I saw that. Hugely confusing if you try to debug stuff and comment out type annotations without effect :) Doesn't help with Flow being so weak, though. If you want to have a look, frain is in quite a good shape now. Just:

1. check out https://github.com/adrianheine/prosemirror/tree/frain
2. npm i getdocs
3. TARGET=flow sh frain.sh
4. npm i typescript
5. TARGET=typescript sh frain.sh

**Comment by marijnh** [comment]
What do you mean by Flow being so weak? It doesn't check enough?

**Comment by marijnh** [comment]
(By the way, I suspect that the changes I'm planning as part of #305 are going to help with getting type checkers to understand what is going on.)

**Pull request closed without merging**